eve_swagger/objects.py

```diff
@@ -16,6 +16,11 @@
 from .definitions import INFO, HOST
 
 
+def _get_scheme():
+    print(app.auth)
```

Comment:
could you please remove this line

Reply:
It's done, and fixed the host url. Everything shoudl be fine now.